templates/corporate/support_request_thanks.html

```diff
@@ -0,0 +1,18 @@
+{% extends "zerver/portico_signup.html" %}
+
+{% block portico_content %}
+<div class="register-account flex full-page">
+    <div class="center-block new-style">
+        <div class="pitch">
+            <h1>Thanks for getting in touch with us!</h1>
+            <p>Your message has been successfully submitted.</p>
```

Comment:
Maybe replace with, "We will be in touch with you soon," or something along those lines?

Reply:
Yeah, that change.